var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -58,23 +59,40 @@ class Gromacs(CMakePackage):
         description='Produces a double precision version of the executables')
     variant('plumed', default=False, description='Enable PLUMED support')
     variant('cuda', default=False, description='Enable CUDA support')
+    variant('nosuffix', default=False, description='Disable default suffixes')
     variant('build_type', default='RelWithDebInfo',
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel',
                     'Reference', 'RelWithAssert', 'Profile'))
-    variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
+    variant('rdtscp', default=True,
+            description='Enable RDTSCP instruction usage')
     variant('mdrun_only', default=False,
             description='Enables the build of a cut-down version'
             ' of libgromacs and/or the mdrun program')
-    variant('openmp', default=True, description='Enables OpenMP at configure time')
-    variant('double_precision', default=False, description='Enables a double-precision configuration')
-    variant('hwloc', default=True, description='Use the hwloc portable hardware locality library')
+    variant('openmp', default=True,
+            description='Enables OpenMP at configure time')
+    variant('double_precision', default=False,
+            description='GMX_RELAXED_DOUBLE_PRECISION for Fujitsu PRIMEHPC')
+    variant('hwloc', default=True,
+            description='Use the hwloc portable hardware locality library')
+    variant('fft', default='fftw3', description="The FFT library to use",
+            values=('fftw3', 'mkl'))
```

Comment:
Notice there exists a `fftw-api@3` virtual package, so that users can have a uniform way to specify their preferred fftw implementation
```suggestion
    
```